Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -56,6 +56,12 @@ protected override void OnEnable()
         {
             base.OnEnable();
 
+            if (target == null)
```

Comment:
I think the class hierarchy goes:

MixedRealityToolkitConfigurationProfileInspector -> BaseMixedRealityToolkitConfigurationProfileInspector -> BaseMixedRealityProfileInspector

Since BaseMixedRealityToolkitConfigurationProfileInspector doesn't override OnEnable, in theory this behavior should already exist from the code in BaseMixedRealityProfileInspector.

Unless this is being done because you're afraid that future people who override BaseMixedRealityToolkitConfigurationProfileInspector  will not call base.OnEnable?

Reply:
What I mean to say is, it seems like this instance of the code isn't necessary because it's handled in the base class. If it's being done again in a subclass (and only one of them!) it would be great to understanding the motivation behind it.

Reply:
Had to short circuit the sub class.

Reply:
Ah doh, yes that's obvious now